include/zephyr/pm/policy.h

```diff
@@ -137,38 +137,37 @@ void pm_policy_state_lock_put(enum pm_state state, uint8_t substate_id);
  */
 bool pm_policy_state_lock_is_active(enum pm_state state, uint8_t substate_id);
 
-
 /**
  * @brief Register an event.
  *
  * Events in the power-management policy context are defined as any source that
  * will wake up the system at a known time in the future. By registering such
  * event, the policy manager will be able to decide whether certain power states
  * are worth entering or not.
- * CPU is woken up before the time passed in cycle to prevent the event handling
- * latency
  *
- * @note It is mandatory to unregister events once they have happened by using
- * pm_policy_event_unregister(). Not doing so is an API contract violation,
- * because the system would continue to consider them as valid events in the
- * *far* future, that is, after the cycle counter rollover.
+ * CPU is woken up before the time passed in cycle to minimize event handling
+ * latency. Once woken up, the CPU will be kept awake until the event has been
+ * handled, which is signaled by pm_policy_event_unregister().
```

Comment:
Is it also valid to call `pm_policy_event_update` instead of unregister/register, or is that only allowed before the event time has passed?

Reply:
It is also valid to call pm_policy_event_update instead of unregister/register :) I will update the desc

Reply:
Updated to:

```
 * CPU is woken up before the time required for minimized event handling latency.
 * to be in effect. Once woken up, the CPU will be kept awake until the event has
 * been handled, which is signaled by pm_policy_event_unregister() or moving event
 * into the future using pm_policy_event_update().
 ```